Common/GPU/Vulkan/VulkanRenderManager.cpp

```diff
@@ -171,8 +173,12 @@ void VKRGraphicsPipeline::DestroyVariants(VulkanContext *vulkan, bool msaaOnly)
 
 void VKRGraphicsPipeline::QueueForDeletion(VulkanContext *vulkan) {
 	DestroyVariants(vulkan, false);
```

Comment:
We could just delete immediately, but we'd need the VulkanContext or device in the callback.  Maybe it's not terrible for all deleter callbacks to get the context...

-[Unknown]

Reply:
Fixed in #16762